### PR TITLE
more documentation clean up

### PR DESCRIPTION
**Pull request by VVelox**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

Went through making more documentation happier with mdl.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
We should add some markdown linter to Travis, I can take a look at it.

**Comment by VVelox** [comment]
@Jellyfrog Awesome. I would suggest telling it to ignore long lines and HTML. More than a few of the code snippets in the docs will trigger the long lines bit and for the most part little can be done about those. As to HTML, that will trigger on some of the integration and alerting docs and there is nothing that can be done about that as they are all in code snippets for showing HTML examples.

**Comment by Jellyfrog** [comment]
Running https://github.com/markdownlint/markdownlint on the whole project, with this PR as a base:
```
   5514  MD013 Line length
   2482  MD004 Unordered list style
   1887  MD005 Inconsistent indentation for list items at the same level
    705  MD009 Trailing spaces
    667  MD022 Headers should be surrounded by blank lines
    641  MD024 Multiple headers with the same content
    560  MD032 Lists should be surrounded by blank lines
    514  MD006 Consider starting bulleted lists at the beginning of the line
    408  MD033 Inline HTML
    395  MD003 Header style
    271  MD012 Multiple consecutive blank lines
    250  MD010 Hard tabs
    152  MD046 Code block style
    142  MD029 Ordered list item prefix
    138  MD034 Bare URL used
    122  MD036 Emphasis used instead of a header
    119  MD031 Fenced code blocks should be surrounded by blank lines
     85  MD026 Trailing punctuation in header
     82  MD002 First header should be a top level header
     77  MD025 Multiple top level headers in the same document
     73  MD001 Header levels should only increment by one level at a time
     50  MD018 No space after hash on atx style header
     31  MD014 Dollar signs used before commands without showing output
     18  MD007 Unordered list indentation
      9  MD028 Blank line inside blockquote
      4  MD027 Multiple spaces after blockquote symbol
      3  MD030 Spaces after list markers
      2  MD038 Spaces inside code span elements
      1  MD039 Spaces inside link text
```
We can at least ignore:
```
   5514  MD013 Line length
    408  MD033 Inline HTML
```
And some are really easy to fix, like trailing spaces.

File: doc/Extensions/Smokeping.md [hunk 1]
```diff
@@ -181,7 +197,7 @@ After that you need to create password for your user
 htpasswd -c /etc/nginx/.htpasswd USER
 ```
 
-You can verify your user and password with 
+You can verify your user and password witt
```
Comment (Jellyfrog) [inline doc/Extensions/Smokeping.md]: Spelling

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-56-release-changelog-september-2019/9671/1

**Pull request merged**